derived_images/portus/README.md

```diff
@@ -124,12 +124,16 @@ Security related settings:
   * `PORTUS_CHECK_SSL_USAGE_ENABLED`: Set this to `false` if you want to disable
     SSL altogether.
 
-Database releated settings:
-
-  * `PORTUS_PRODUCTION_HOST`: the host running the MariaDB database.
-  * `PORTUS_PRODUCTION_USERNAME`: the database user to be used.
-  * `PORTUS_PRODUCTION_PASSWORD`: the password of the database user.
-  * `PORTUS_PRODUCTION_DATABASE`: the name of the Portus database.
+Database releated settings (se [configuring the database](http://port.us.org/docs/database.html) for details):
+
+  * `PORTUS_DB_ADAPTER`: databasetype. Legal values are postgresql or mysql2. Default is mysql2
```

Comment:
Typo: "databasetype" -> "database type". Moreover, instead of "Legal values", let's say "Supported values".

Reply:
"Supported values" are better, for sure :-)